Exiled.Events/Patches/Events/Player/TriggeringTesla.cs

```diff
@@ -27,71 +27,71 @@ namespace Exiled.Events.Patches.Events.Player
     [HarmonyPatch(typeof(TeslaGateController), nameof(TeslaGateController.FixedUpdate))]
     internal static class TriggeringTesla
     {
-        private static bool Prefix(TeslaGateController __instance)
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator generator)
         {
-            try
+            List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Pool.Get(instructions);
+
+            Label retLabel = generator.DefineLabel();
+
+            const int offset = 1;
+
+            // remove the reference hub Foreach
+            int index = newInstructions.FindIndex(
+                instruction => instruction.Calls(PropertyGetter(typeof(ReferenceHub), nameof(ReferenceHub.AllHubs))));
+
+            newInstructions.RemoveRange(index, newInstructions.FindIndex(x => x.opcode == OpCodes.Endfinally) + offset - index);
```

Comment:
Comment what you're removing.
```suggestion
            newInstructions.RemoveRange(index, newInstructions.FindIndex(i => i.opcode == OpCodes.Endfinally) + offset - index);
```